splito/lohi/_lo.py

```diff
@@ -0,0 +1,160 @@
+from rdkit import Chem, DataStructs
+from rdkit.Chem import AllChem
+import numpy as np
+from tqdm import tqdm
+
+from ._utils import get_similar_mols
+
+class LoSplitter():
+    def __init__(self, threshold=0.4, min_cluster_size=5, max_clusters=50, std_threshold=0.60):
+        """
+        Parameters:
+            threshold -- molecules with a similarity higher than this value are considered similar.
+            min_cluster_size -- number of molecules per cluster.
+            max_clusters -- maximum number of selected clusters. The remaining molecules go to the training set.
+            std_threshold -- Lower bound of the acceptable standard deviation for a cluster. It should be greater than measurement noise.
+                            If you're using ChEMBL-like data, set it to 0.60 for logKi and 0.70 for logIC50.
+                            Set it lower if you have a high-quality dataset.
+        """
+        self.threshold = threshold
+        self.min_cluster_size = min_cluster_size
+        self.max_clusters = max_clusters
+        self.std_threshold = std_threshold
+
+    def split(self, smiles, values, verbose=1):
+        """
+        Split the dataset of smiles and their continuous values.
+
+        Parameters:
+            smiles -- list of smiles.
+            values -- list of their continuous activity values.
+            verbose -- set to 0 to turn off progressbar.
+
+        Returns:
+            train_idx -- list of train indices.
+            clusters_idx -- list of lists of cluster indices.
+        """
+        if not isinstance(smiles, np.ndarray):
+            smiles = np.array(smiles)
+        if smiles.shape != np.unique(smiles).shape:
+            raise ValueError("Remove duplicates from your smiles")
+
+        if not isinstance(values, np.ndarray):
+            values = np.array(values)
+
+        train_idx, clusters_idx = self._select_distinct_clusters(
+            smiles, values, verbose
+        )
+        train_idx = list(train_idx)
+
+        # Move one molecule from each cluster to the training set
+        leave_one_clusters = []
+        for cluster in clusters_idx:
+            train_idx.append(cluster[-1])
+            leave_one_clusters.append(cluster[:-1])
+        
+        if not leave_one_clusters:
+            print('No clusters were found. Was your std_threshold too constrained?')
```

Comment:
Please use the [`loguru` logger](https://loguru.readthedocs.io/en/stable/index.html) instead of a print statement: 

```suggestion
            loguru.warninig('No clusters were found. Was your std_threshold too constrained?')
```

Reply:
Will do. Also please check lines #67 and #100. 
1) Should I use verbose flag for the logger, or `logger.info` would be a better solution?
2) Is it ok to use tqdm?

Reply:
`logger` has various levels already, see [here](https://loguru.readthedocs.io/en/stable/api/logger.html#levels). You can configure the level globally to control the verbosity. You would then only use the `verbose` flag for the `tqdm` progress bar. Using `tqdm` is not a problem!